client/my-sites/people/people-profile/index.jsx

```diff
@@ -219,8 +223,26 @@ const PeopleProfile = ( { siteId, type, user, invite } ) => {
 				</>
 			);
 		}
+		if ( p2Guests?.guests && p2Guests.guests.includes( user?.linked_user_ID ?? user?.ID ) ) {
+			p2GuestBadge = (
+				<>
+					<div className="people-profile__role-badge role-p2-guest">
+						{ translate( 'Guest', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **81** times:
`translate( 'Guest' )` *ES Score: 12*


:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?